airbyte-integrations/bases/base-normalization/integration_tests/test_ephemeral.py

```diff
@@ -62,7 +62,11 @@ def setup_test_path(request):
 
 @pytest.mark.parametrize("column_count", [1500])
 @pytest.mark.parametrize("integration_type", list(DestinationType))
-def test_destination_supported_limits(integration_type: DestinationType, column_count: int, setup_test_path):
+def test_destination_supported_limits(integration_type: DestinationType, column_count: int):
+    if integration_type == DestinationType.MYSQL:
+        # In MySQL, the max number of columns is limited by row size (8KB),
+        # not by absolute column count. It is way fewer than 1500.
+        return
```

Comment:
Not to be solved in this PR but we should probably handle the validity of rows (in terms of size) in the `destination-mysql` and log some warning/errors or reject such records?

About 250 columns, seems to be really low, especially for sources like hubspot/salesforce and they could easily run into exceptions there?

Reply:
The max number of columns is related to the size of each column. In the test, we are creating string columns, which corresponds to `char` columns in mysql and each column is relatively large. That's why as few as 250 columns can fail the test. In reality, not every column will be of type `char`, and the max is 4096 in mysql.